chain/eth_transaction_hash_lookup.go

```diff
@@ -0,0 +1,162 @@
+package chain
```

Comment:
This belongs in its own package (or at the very least doesn't belong in the root chain package)